pinot-core/src/main/java/org/apache/pinot/core/realtime/impl/kafka/KafkaPartitionLevelConsumer.java

```diff
@@ -38,14 +38,21 @@
 public class KafkaPartitionLevelConsumer extends KafkaConnectionHandler implements PartitionLevelConsumer {
   private static final Logger LOGGER = LoggerFactory.getLogger(KafkaPartitionLevelConsumer.class);
 
+  private final int _fetchRequestMinBytes;
+  private final int _fetchRequestSize;
+
   public KafkaPartitionLevelConsumer(String clientId, StreamConfig streamConfig, int partition) {
     super(clientId, streamConfig, partition, new KafkaSimpleConsumerFactoryImpl());
+    this._fetchRequestSize = getkafkaLowLevelStreamConfig().getKafkaFetcherSizeBytes();
```

Comment:
Can you make the variable name consistent? `fetchRequestSize vs fetcherSizeBytes`